.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: "Close stale issues"
+on:
+  schedule:
+    # hourly
+    - cron: "0 * * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          only-labels: "Response Requested"
+          stale-issue-message: "Greetings! It looks like this issue hasn't been active in longer than a week. Because it has been a while since the last update on this, and in the absence of more information, we will be closing this issue soon. Please feel free to provide a comment or add an upvote to prevent automatic closure, or if the issue is already closed, please feel free to open a new one."
```

Comment:
> add an upvote to prevent automatic closure

Can you confirm that adding a reaction like thumbs-up to an issue does indeed change the `updated_at` field for the issue in the API?
https://developer.github.com/v3/issues/#get-an-issue

Reply:
I tested it out and you're right, a reaction doesn't change the `updated_at` field. Fixed in the latest revision.